tests/foreman/api/test_contentmanagement.py

```diff
@@ -123,6 +124,51 @@ def test_positive_sync_repos_with_lots_files(self):
         with self.assertNotRaises(TaskFailedError):
             repo.sync()
 
+    @tier4
+    def test_positive_sync_kickstart_repo(self):
+        """No encoding gzip errors on kickstart repositories
+        sync.
+
+        :id: dbdabc0e-583c-4186-981a-a02844f90412
+
+        :expectedresults: No encoding gzip errors present in /var/log/messages.
+
+        :CaseLevel: Integration
+
+        :steps:
+
+            1. Sync a kickstart repository.
+            2. After the repo is synced, change the download policy to
+                immediate.
+            3. Sync the repository again.
+            4. Assert that no errors related to encoding gzip are present in
+                /var/log/messages.
+            5. Assert that sync was executed properly.
+
+        :CaseComponent: Pulp
+
+        :BZ: 1687801
+        """
+        org = entities.Organization().create()
+        product = entities.Product(organization=org).create()
+        repo = entities.Repository(
+            product=product,
+            url=CUSTOM_KICKSTART_REPO
+        ).create()
+        repo.sync()
+        repo.download_policy = 'immediate'
+        repo = repo.update(['download_policy'])
+        call_entity_method_with_timeout(repo.sync, timeout=600)
+        result = ssh.command(
+            'grep pulp /var/log/messages | grep failed | grep encoding | grep gzip'
+        )
+        self.assertEqual(result.return_code, 1)
+        self.assertEqual(len(result.stdout), 0)
+
+        repo = repo.read()
+        self.assertGreater(repo.content_counts['package'], 0)
+        self.assertGreater(repo.content_counts['package_group'], 0)
+        self.assertGreater(repo.content_counts['rpm'], 0)
```

Comment:
flake8 not well 
`./tests/foreman/api/test_contentmanagement.py:173:1: E302 expected 2 blank lines, found 1 `

Reply:
`make can-i-push` does not detect this. Updated.